### PR TITLE
Validate email of person for format and domain

### PR DESCRIPTION
**Pull request by novotnyjakub**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -150,6 +150,8 @@ en:
     errors:
       messages:
         blank: "is required"
+        invalid_format: "isn't valid"
+        invalid_domain: "you have entered can't be used to access People Finder"
```
Comment (threedaymonk) [inline config/locales/en.yml]: Let’s use proper apostrophes: `’` rather than `'`.

Comment (novotnyjakub) [inline config/locales/en.yml]: I see that we have them mixed throughout the file. Is this for sure the correct one? Cause I'll fix all of them if that's the case.

Comment (threedaymonk) [inline config/locales/en.yml]: In that case, let’s fix all the apostrophes as a separate commit.

**Comment by threedaymonk** [comment]
Rebased and merged.

**Pull request closed without merging**